supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/GenomicsFactory.scala

```diff
@@ -88,15 +88,18 @@ case class GenomicsFactory(applicationName: String, authMode: GoogleAuthMode, en
 
       def networkFromLabels(vpcConfig: VirtualPrivateCloudConfiguration, projectLabels: ProjectLabels): Network = {
         val networkLabelOption = projectLabels.labels.find(l => l._1.equals(vpcConfig.name))
+        val subnetworkLabelOption = vpcConfig.subnetwork.flatMap(s => projectLabels.labels.find(l => l._1.equals(s)))
 
-        networkLabelOption match {
-          case Some(networkLabel) =>
-            new Network()
+        (networkLabelOption, subnetworkLabelOption) match {
```

Comment:
No need to add `subnetworkLabelOption` to the match if you just use `_` in both `case`s

Reply:
Hmm that's right. I guess I missed that while refactoring it. I have removed it. Thank you.